src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -133,7 +115,15 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             },
             refreshView: {
                 funcName: "fluid.slidingPanel.refreshView",
-                args: ["{that}"]
+                args: ["{that}", "{that}.panelId"]
+            },
+            setShowText: {
```

Comment:
This should be achieved via model values rather than a separate invoker per element